receiver/dcgmreceiver/scraper_gpu_test.go

```diff
@@ -28,13 +28,14 @@ import (
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/receiver"
 	"go.uber.org/zap/zaptest"
 
 	"github.com/GoogleCloudPlatform/opentelemetry-operations-collector/receiver/dcgmreceiver/testprofilepause"
 )
 
 func TestScrapeWithGpuPresent(t *testing.T) {
-	settings := componenttest.NewNopReceiverCreateSettings()
+	var settings receiver.CreateSettings
```

Comment:
`componenttest.NewNopReceiverCreateSettings()` was deprecated in collector v0.69.0